paper_on_queues_and_stack_structures.md

```diff
@@ -0,0 +1,21 @@
+# Introduction to Queues and Stack Structures  
+Queues and stack are data structures that differ when it comes to the modification of data. The principle for queues is the data that is first in, gets out first. For stacks, the principle is last in, first out. 
+A queue can be visualized as a sequence of rectangular boxes arranged from left to right with random objects (data) in each one.  With a stack, the same rectangle boxes are arranged bottom to top similar to a stack of paper. Queues and stack structures have their own trade-offs and they both play vital roles in the computer science.
```

Comment:
Delete "random objects" and just say "data."
Change "rectangle" to "rectangular"
Comma after "bottom to top"